bcda/auth/provider.go

```diff
@@ -20,52 +18,31 @@ const (
 	SSAS  = "ssas"
 )
 
-var providerName = Alpha
+var providerName = SSAS
 var repository models.Repository
 
 func init() {
 	log.SetFormatter(&log.JSONFormatter{})
 	log.SetReportCaller(true)
-	SetProvider(strings.ToLower(conf.GetEnv(`BCDA_AUTH_PROVIDER`)))
+	// SetProvider(strings.ToLower(conf.GetEnv(`BCDA_AUTH_PROVIDER`)))
 
 	repository = postgres.NewRepository(database.Connection)
 }
 
 func SetProvider(name string) {
```

Comment:
Im assuming this method is probably not necessary anymore since we the provider name is hardcoded on line 21 and we dont support other providers anymore.

Reply:
Also yes. Future PR will include revamping the test.